package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "setup-dbs": "psql -f ./db/setup.sql",
     "seed": "node ./db/seeds/run-seed.js",
-    "test": "jest",
+    "test": "NODE_ENV='test' jest",
```

Comment:
you don't need this. When you run your test file, the NODE_ENV will be set to "test" automatically